1/contrib/openshift/configuration/jobs/OpenShift Sample/config.xml

```diff
@@ -22,7 +22,7 @@
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
       <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <bldCfg>frontend</bldCfg>
+      <bldCfg>ruby-sample-build</bldCfg>
```

Comment:
/me confused.

The Jenkins tutorial say to create the sample app so I think QE went ahead and create that instead of application-template.json. The sample is ruby but this app is nodejs :-) That means all these changes are not needed and we just need to instruct QE to use the application-template.json instead of sample-app template.